tests/testthat/test-survival-tune-bayes.R

```diff
@@ -64,33 +65,81 @@ test_that("Bayesian tuning survival models with static metric", {
       initial = init_grid_static_res
     )
 
+  # ------------------------------------------------------------------------------
+  # test structure of results
+
   expect_false(".eval_time" %in% names(bayes_static_res$.metrics[[1]]))
   expect_equal(
     names(bayes_static_res$.predictions[[1]]),
     c(".pred_time", ".row", "tree_depth", "event_time", ".config")
   )
 
+  # ------------------------------------------------------------------------------
+  # test autoplot
+
   expect_snapshot_plot(
     print(autoplot(bayes_static_res)),
-    "static-metric-bayes-search"
+    "stc-bayes"
   )
 
   expect_snapshot_plot(
     print(autoplot(bayes_static_res, type = "marginals")),
-    "static-bayes-search-with-two-time-points-marginals"
+    "stc-bayes-2-times-marg"
   )
   expect_snapshot_plot(
     print(autoplot(bayes_static_res, type = "performance")),
-    "static-bayes-search-with-two-time-points-performance"
+    "stc-bayes-2-times-perf"
   )
+
+  # ------------------------------------------------------------------------------
+  # test metric collection
+
+  metric_sum <- collect_metrics(bayes_static_res)
+  exp_metric_sum <-
+    structure(
+      list(
+        tree_depth = numeric(0),
+        .metric = character(0),
+        .estimator = character(0),
+        mean = numeric(0),
+        n = integer(0),
+        std_err = numeric(0),
+        .config = character(0),
+        .iter = integer(0)
+      ),
+      row.names = integer(0),
+      class = c("tbl_df", "tbl", "data.frame"))
```

Comment:
I find it cleaner to just create the tibble normally instead of using `structure()`. Whenever I see `structure()` I do a double take to make sure it isn't doing something unusual 

```suggestion
  exp_metric_sum <- tibble(
    tree_depth = numeric(0),
    .metric = character(0),
    .estimator = character(0),
    mean = numeric(0),
    n = integer(0),
    std_err = numeric(0),
    .config = character(0),
    .iter = integer(0)
  )
```

Reply:
I only use `structure()` because it is produced by `dput()`. `tribble_paste()` is better but chokes on list columns.

Reply:
or `ctrl + shift + R`


https://github.com/tidymodels/extratests/assets/14034784/3a2a9dcf-ada8-4ac4-868f-4e4c1b0f2228